app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -5,7 +5,7 @@ function initializeCommentsPage() {
     toggleCodeOfConduct();
     var commentableId = document.getElementById('comments-container').dataset.commentableId;
     var commentableType = document.getElementById('comments-container').dataset.commentableType;
-    commentableIdList = commentableId.split(",");
+    var commentableIdList = commentableId.split(",");
     var f = (function() {
       for (var i = 0; i < commentableIdList.length; i++) {
         (function(i){
```

Comment:
I don't get it. This file shouldn't be ignored by eslint?https://github.com/thepracticaldev/dev.to/blob/c8b6c9c09504ddf77f00ab65b5a8859ea7250b92/app/assets/javascripts/.eslintrc.js#L9
If i remove this change the tests will fail.

Reply:
My question is: why this file is in the PR at all? :)

Reply:
Because all these files will be grouped on a single file when loaded on browser. We're using `use strict` on `initNotification` and lines like this https://github.com/thepracticaldev/dev.to/blob/b42dfed84491f582d77e2891a3c2d16a151a3851/app/assets/javascripts/initializers/initializeCommentsPage.js.erb#L8
will return undefined because the `use_strict` don't allow global var, and the tests will fail.